### ISSUE TITLE
Models occasionally render as black

### ISSUE THREAD

**Issue by SK83RJOSH** [KiwiOrg/Kiwi#115]
While making models for the golf game mode I've come across a lot of instances where objects render totally black. It appears like the material is not correctly applied or similar, as they have identical materials:
![image](https://user-images.githubusercontent.com/1371787/219951690-fa413f34-0d5a-442a-b9a1-9ca76ac5a0e5.png)

In game, the ball, arrow, and the line all use the same material; but are separate models to avoid needing to setup complex hierarchies. Despite this, the triangular indicator is rendered as black:
![image](https://user-images.githubusercontent.com/1371787/219951848-5e521a39-afd3-401c-af05-d7b89d443a8e.png)

I've had this happen with other models as well, and at times, without clearing the cache or making any changes at all, it would have the correct material the next run.

**Comment by SK83RJOSH** [comment]
[indicator_arrow.zip](https://github.com/KiwiOrg/Kiwi/files/10777012/indicator_arrow.zip)

**Comment by philpax** [comment]
Can you try turning on the various rendering debug views and seeing if there's anything amiss? Wondering if it's a normal issue

**Comment by SK83RJOSH** [comment]
Disable Shading:
![image](https://user-images.githubusercontent.com/1371787/220071276-715e9218-eb53-4662-9b3e-e7afd1b95deb.png)

Normals:
![image](https://user-images.githubusercontent.com/1371787/220071308-c6a5834a-8726-470e-b6dc-08814bc40e98.png)

Metallic Roughness:
![image](https://user-images.githubusercontent.com/1371787/220071368-9e91b7e3-f342-4205-94ea-61c45d8b720a.png)

**Comment by philpax** [comment]
After recreating the mesh, was this still an issue?

**Comment by SK83RJOSH** [comment]
> After recreating the mesh, was this still an issue?

Nope, but I still think this issue is relevant given the mesh having correct normals and materials when imported into blender. ^^

**Comment by ten3roberts** [comment]
I am having a similar issue, where the `third_person_example` is black on `debug` but randomly colored on `release`

Not sure if it is only a Linux issue
